tekton/resources.yaml

```diff
@@ -20,7 +20,7 @@ spec:
    - name: type
      value: gcs
    - name: location
-     value: gs://tekton-releases # REPLACE with your own bucket
```

Comment:
You mentioned the importance of this in person, let's add this comment back in 😄

Reply:
done